drivers/soundwire/intel.c

```diff
@@ -519,13 +518,9 @@ static int intel_shim_sync_go_unlocked(struct sdw_intel *sdw)
 	 */
 	sync_reg |= SDW_SHIM_SYNC_SYNCGO;
 
-	ret = intel_clear_bit(shim, SDW_SHIM_SYNC, sync_reg,
-			      SDW_SHIM_SYNC_SYNCGO);
-
-	if (ret < 0)
-		dev_err(sdw->cdns.dev, "SyncGO clear failed: %d\n", ret);
+	intel_writel(shim, SDW_SHIM_SYNC, sync_reg);
```

Comment:
@plbossart, I'm confused. If the `SHIM_SYNC::SYNC_GO` bit is read-only then writing to it should not have any effect, no?
Probably the SYNCGO bit is a write-only and always reads 0 type of bit (which is an interesting design decision)?